notification/tests.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from notification.models import Notification
+
+
+@pytest.mark.django_db
+class TestNotification:
+    def test_create_notification(self, notification):
+        assert isinstance(notification, Notification)
+
+    def test_persisted_notification_in_database(self, notification):
+        assert notification in Notification.objects.all()
+
+    def test_delete_notification_from_database(self, notification):
+        notification.delete()
+        with pytest.raises(Notification.DoesNotExist):
+            Notification.objects.get(pk=notification.pk)
+
+    def test_mark_as_read_notification(self, notification):
+        notification.mark_notification_as_read()
+        assert notification.is_read is True
```

Comment:
I think we better check it was false before to see that we made a difference.

Reply:
Added this check line to this test.